src/main/blackbox/blackbox.c

```diff
@@ -1045,6 +1045,13 @@ static void writeInterframe(void)
     blackboxLoggedAnyFrames = true;
 }
 
+// for gcc9 (and maybe later), we warn on (mainly harmless) unaligned packed structure member pointer access
+// minimise the effect to the affected functions and compiler versions.
+#if (__GNUC__ >= 9)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Waddress-of-packed-member"
```

Comment:
Shouldn't we move this to `platform.h` to cover all the code? Or we want to keep the warning for other files?

Reply:
In hindsight, that is probably the most logical thing to do if we accept that (a) this is harmless and (b) we've been implicitly doing it forever.

If new code generates the warning,  it's most likely because we have another good reason to pack the structures, and we'll want to ignore the warning.

I'll update the PR accordingly.